api/storage_service.cc

```diff
@@ -615,6 +615,7 @@ void set_storage_service(http_context& ctx, routes& r, sharded<service::storage_
         if (column_families.empty()) {
             column_families = map_keys(ctx.db.local().find_keyspace(keyspace).metadata().get()->cf_meta_data());
         }
+        apilog.info("force_keyspace_compaction: keyspace={} tables={}", keyspace, column_families);
         return ctx.db.invoke_on_all([keyspace, column_families] (replica::database& db) -> future<> {
             auto table_ids = boost::copy_range<std::vector<table_id>>(column_families | boost::adaptors::transformed([&] (auto& cf_name) {
```

Comment:
user-initiated compactions are logged by compaction manager. 

for example in compaction manager we have:
        cmlog.info0("User initiated compaction started on behalf of {}.{}", t->schema()->ks_name(), t->schema()->cf_name());

so logging it here may be redundant.

Reply:
okay

Reply:
Demoted to debug level in v2

Reply:
But now all the other operations will have logs on the API level, except this. This could be terribly confusing for someone getting used to info logs on API operations.

Reply:
@denesb Do you thinking printing both message in info level would be better (it should be a rare event so that shouldn't spam the log)

Reply:
I actually think an info log on every API request is a good idea, it serves as an audit log and as feedback to the user that their command was received and processed. I'm not too worried about the duplicate messages here as like you said this is a rare event. Also the "user-requested" in the compaction manager simply means outside code, not necessarily the user who sits at the desk. So it is not strictly a full duplicate message.